From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cheng Zhao <zcbenz@gmail.com>
Date: Thu, 20 Sep 2018 17:48:16 -0700
Subject: no_cache_storage_check.patch

Do not check for unique origin in CacheStorage, in Electron we may have
scripts running without an origin.

diff --git a/content/browser/cache_storage/legacy/legacy_cache_storage.cc b/content/browser/cache_storage/legacy/legacy_cache_storage.cc
index e4d567bc532aed483f462a14112d88e6897e23f1..c6809ef67b312d3592e43fb7f0d02f482ad18737 100644
--- a/content/browser/cache_storage/legacy/legacy_cache_storage.cc
+++ b/content/browser/cache_storage/legacy/legacy_cache_storage.cc
@@ -108,7 +108,7 @@ class LegacyCacheStorage::CacheLoader {
         cache_storage_(cache_storage),
         origin_(origin),
         owner_(owner) {
-    DCHECK(!origin_.opaque());
+    // DCHECK(!origin_.opaque());
   }
 
   virtual ~CacheLoader() {}
